projects/plugins/jetpack/extensions/shared/jetpack-plugin-sidebar.js

```diff
@@ -16,6 +17,19 @@ const { Fill, Slot } = createSlotFill( 'JetpackPluginSidebar' );
 
 export { Fill as default };
 
+/**
+ * Open Jetpack plugin sidebar by default when URL includes query param.
+ */
+function openJetpackSidebar() {
```

Comment:
This also works as a component with hooks that we render in `registerPlugin`, but it seemed strange to render an empty component.

Reply:
```suggestion
function openJetpackSidebar() {
	if ( getQueryArg( window.location.search, 'jetpackSidebarIsOpen' ) !== 'true' ) {
		return;
	}

```